ports/kf5iconthemes/vcpkg.json

```
@@ -0,0 +1,19 @@
+{
+  "name": "kf5iconthemes",
+  "version-string": "5.75.0",
```

```suggestion
  "version": "5.75.0",
```
The package versions should be more proper. Please see https://github.com/microsoft/vcpkg/blob/master/docs/specifications/versioning.md#22-package-versions